src/lang/en-US.json

```diff
@@ -70,6 +70,7 @@
   "app_blog": "Blog",
   "app_translate": "Translate",
   "app_donate": "Donate",
+  "app_netlify": "Deploys by Netlify",
```

Comment:
Hosted by?

Reply:
Deploys by Netlify is the text they use on their badges: https://www.netlify.com/press/